README.md

```diff
@@ -384,6 +386,18 @@ voucherify.segments.delete(id);
 
 ---
 
+### Custom Events API
+Methods are provided within `voucherify.events.*` namespace.
+
+- [Custom events](#custom-events)
```

Comment:
this should rather be `[Create Event] (#create-event)` and the following `#### [Create custom event]` changed to `#### [Create Event]` and lastly the link to docs from `[Custom Events]` to `[Create Event]`